src/audio/fft/fft.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2017 Intel Corporation. All rights reserved.
+ *
+ * Author: Amery Song <chao.song@intel.com>
+ *	   Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+#include <sof/common.h>
+
+#define FFT_SIZE_MAX	1024
+
+struct icomplex32 {
```

Comment:
I would argue we should make complex math a subsection of the math lib

Reply:
> I would argue we should make complex math a subsection of the math lib

I am hesitated to do this as here it is integer real/imag and not that generic float complex for math lib.

Reply:
I know, but we dont support float yet so this seems like best place to put it if anyone else needs it

Reply:
@cujomalainey let me consider moving it in next version, thanks.